apps/kitchen-sink/src/ensemble/screens/widgets.yaml

```diff
@@ -54,10 +54,37 @@ View:
                       value: option1
                     - label: Option 2
                       value: option2
-        - Card:
-            styles:
-              maxWidth: unset
-              width: unset
+              - Dropdown:
+                  label: With item template
+                  item-template:
+                    data:
+                      [
+                        { name: "hello", value: 11 },
+                        { name: "buyyy", value: 22 },
+                      ]
+                    name: product
+                    value: ${product.value}
+                    template:
+                      Text:
```

Comment:
Does this follow the flutter sides contract? I think we also need to bind the value

Reply:
Yes, I have seen the definition of flutter implementation for the item-template inside the dropdown. they are using this way to passing and evaluate the value on each option